guvnor-ala/guvnor-ala-build-maven/src/main/java/org/guvnor/ala/build/maven/config/MavenBuildConfig.java

```diff
@@ -34,14 +34,14 @@
     */
     default List<String> getGoals() {
         final List<String> result = new ArrayList<>();
-        result.add( "package" );
+        result.add("package");
         return result;
     }
 
     default Properties getProperties() {
         final Properties result = new Properties();
-        result.setProperty("failIfNoTests", "false" );
+        result.setProperty("failIfNoTests",
+                           "false");
```

Comment:
@manstis @jhrcek @hasys Guys, this is a result of an error in the IDEA config introduced [here](https://github.com/kiegroup/droolsjbpm-build-bootstrap/commit/60e3f5bd9f99b925286a74df71de3cb2e35dab08#diff-9d663bfb476dd8a86f4f213da123bfa5R76). Method call arguments and methods declaration parameters should only be wrapped when the line is too long. It can be achieved by using the `Wrap if long` option in IDEA. Currently it is set to `Wrap always`, which is unnecessary and wrong in my opinion. That is not required nor recommended by [Java Code Conventions](http://www.oracle.com/technetwork/java/codeconventions-150003.pdf) mentioned by Michael in [PR#358](https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/358#issuecomment-272396800). I also wonder about @ge0ffrey's opinion and if he intended to approve the `Wrap always` option or `Wrap if long`.